pkgs/applications/editors/manuskript/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, pkgs, fetchFromGitHub
+}:
+let
+  pythonPackages = pkgs.python3Packages;
+in
```

Comment:
probably better to just use python3Packages throughout, and have it passed in as a variable at the top. This way there is no confusion about how pythonPackages usually refers to 2.7.